modules/Bio/EnsEMBL/Compara/PipeConfig/EPOwith2x_conf.pm

```diff
@@ -0,0 +1,302 @@
+        'output_dir'          => '#feature_dir#', # alias
+
+        # Capacities
+        'low_capacity'                  => 10,
+        'map_anchors_batch_size'        => 20,
+        'map_anchors_capacity'          => 2000,
+        'trim_anchor_align_batch_size'  => 20,
+        'trim_anchor_align_capacity'    => 500,
+
+        # Options
+        #skip this module if set to 1
+        'skip_multiplealigner_stats' => 0,
+        # dont dump the MT sequence for mapping
+        'only_nuclear_genome' => 1,
+        # add MT dnafrags separately (1) or not (0) to the dnafrag_region table
+        'add_non_nuclear_alignments' => 1,
+         # batch size of grouped anchors to map
+        'anchor_batch_size' => 500, #mammals
+        #'anchor_batch_size' => 50,  #fish
+         # max number of sequences to allow in an anchor
+        'anc_seq_count_cut_off' => 15,
+
+        # The ancestral_db is created on the same server as the pipeline_db
+        'ancestral_db' => { # core ancestral db
+            -driver   => $self->o('pipeline_db', '-driver'),
+            -host     => $self->o('pipeline_db', '-host'),
+            -port     => $self->o('pipeline_db', '-port'),
+            -species  => $self->o('ancestral_sequences_name'),
+            -user     => $self->o('pipeline_db', '-user'),
+            -pass     => $self->o('pipeline_db', '-pass'),
+            -dbname   => $self->o('dbowner').'_'.$self->o('species_set_name').'_ancestral_core_'.$self->o('rel_with_suffix'),
+        },
+
+
+        # ----- EpoLowCoverage settings ----- #
+
+        'run_gerp' => 1,
+        'gerp_window_sizes'    => [1,10,100,500],         #gerp window sizes
+        
+        'low_epo_mlss_id'  => $self->o('low_epo_mlss_id'),  #mlss_id for low coverage epo alignment
+    	'base_epo_mlss_id' => $self->o('high_epo_mlss_id'), #mlss_id for the base alignment we're topping up
+                                                            # (can be EPO or EPO_LOW_COVERAGE)
+        'max_block_size'  => 1000000,                       #max size of alignment before splitting 
+    
+        #default location for pairwise alignments (can be a string or an array-ref)
+        'pairwise_location' => [ qw(compara_prev lastz_batch_*) ],
+        'epo_db' => $self->pipeline_url(),
+    };
+}
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+    return [
+        @{$self->SUPER::pipeline_create_commands},  # inheriting database and hive tables' creation
+        $self->pipeline_create_commands_rm_mkdir(['work_dir', 'bed_dir', 'feature_dir', 'bl2seq_dump_dir']),
+    ];
+}
+
+
+sub pipeline_wide_parameters {
+    my $self = shift @_;
+    return {
+        %{$self->SUPER::pipeline_wide_parameters},
+
+        'low_epo_mlss_id'  => $self->o('low_epo_mlss_id'),
+    	'high_epo_mlss_id' => $self->o('high_epo_mlss_id'),
+        'base_epo_mlss_id' => $self->o('base_epo_mlss_id'),
+        'species_set_name' => $self->o('species_set_name'),
```

Comment:
I cannot find any runnable or pipeline involved that requires this parameter to be declared as "wide".